src/components/Prompt/Prompt.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useEffect, useState} from 'react';
+
+import {Button, ButtonProps} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+
+import './Prompt.scss';
+
+const b = block('Prompt');
+
+export interface PromptProps {
+    // Prompt message
+    text: string;
+    actions: ButtonProps[];
+    // Unix Timestamp in milliseconds when the Prompt opens
+    openTimestamp?: number;
+    // Milliseconds to remain visible
+    openDuration?: number;
+    className?: string;
+    theme?: 'grey' | 'beige' | 'white';
+}
+
+/**
+ * Popup that appears with text message and button(s) for given `actions`.
+ * Features:
+ *  - Automatically disappears after `openDuration` in milliseconds
+ *  - `openTimestamp` (`Date.now()`) resets the visible duration
+ * @returns {JSX|null}
+ */
+export const Prompt: React.FC<PromptProps> = ({
+    text,
+    actions,
+    className,
+    openTimestamp = 0,
+    openDuration,
+    theme = 'grey',
+}) => {
+    const [open] = useOpenCloseTimer(openTimestamp, openDuration);
+    const mounted = openTimestamp > 0;
+
+    return (
+        <div className={b({theme, open, close: !open, mounted}, className)}>
+            <div className={b('content')}>
```

Comment:
It could be simplier:

`<div><div>{content}</div></div>` -> `<div>{content}</div>` 

```ts
<div className={b('content', {theme, open, close: !open, mounted}, className)}>
    <span className={b('text')}>{text}</span>
    .....
```

Reply:
unfortunately not, the extra `content` wrapper is required for CSS animation, for future extensibility and it follows the same pattern of other existing popups in gravity ui.